cmake/FreecivInstall.cmake

```diff
@@ -169,6 +163,10 @@ if(UNIX AND NOT APPLE)
     message(STATUS "CMAKE_INSTALL_PREFIX changed from the default to /usr.")
   endif()
 
+  install(FILES ${CMAKE_SOURCE_DIR}/data/icons/index.theme
```

Comment:
Icon does not work on Windows. I think this install needs to happen on all platforms. If you install/copy the data/icons/* directory structure including the new file, then windows finds the icon.